main/scons/APKBUILD

```diff
@@ -1,31 +1,29 @@
-# Contributor: 
-# Maintainer: 
+# Contributor:
+# Maintainer:
 pkgname=scons
-pkgver=3.0.5
+pkgver=3.1.1
 pkgrel=0
 pkgdesc="Software construction system"
 url="http://www.scons.org"
 arch="noarch"
 license="MIT"
-# before upgrade to python3, make sure that packages like serf builds with scons with python3
-depends="python2"
-makedepends="python2-dev"
-source="https://downloads.sourceforge.net/$pkgname/$pkgname-$pkgver.tar.gz"
-builddir="$srcdir/$pkgname-$pkgver"
+depends="python3"
```

Comment:
This is somewhat risky. Last time I tried this it broke build for various packages using scons. We need to verify that it doesn't happen again this time. Those are the packages:
```
main/gpsd
main/serf
community/gringo
```